library/std/src/prelude/v1.rs

```diff
@@ -55,7 +55,7 @@ pub use core::prelude::v1::{
 pub use core::prelude::v1::concat_bytes;
 
 // Do not `doc(inline)` these `doc(hidden)` items.
-#[stable(feature = "builtin_macro_prelude", since = "1.38.0")]
+#[unstable(feature = "rustc_encodable_decodable", issue = "none")]
 #[allow(deprecated)]
 pub use core::prelude::v1::{RustcDecodable, RustcEncodable};
```

Comment:
There's a similar reexport in `library/core/src/prelude/v1.rs` that also needs to be marked as unstable.